include/cuco/static_multimap.cuh

```diff
@@ -293,6 +292,35 @@ class static_multimap {
                 KeyEqual key_equal  = KeyEqual{},
                 cudaStream_t stream = 0) const;
 
+  /**
+   * @brief Indicates whether the pairs in the range `[first, last)` are contained in the map.
+   *
+   * Stores `true` or `false` to `(output + i)` indicating if the pair `*(first + i)` exists in
+   * the map.
+   *
+   * ProbeSequence hashers should be callable with both
+   * `std::iterator_traits<InputIt>::value_type::first_type`
+   * and Key type. `std::invoke_result<KeyEqual,
+   * std::iterator_traits<InputIt>::value_type::first_type, Key>`
+   * must be well-formed.
+   *
+   * @tparam InputIt Device accessible random access input iterator
+   * @tparam OutputIt Device accessible output iterator assignable from `bool`
+   * @tparam PairEqual Binary callable type used to compare input pair and slot content for equality
+   *
+   * @param first Beginning of the sequence of pairs
+   * @param last End of the sequence of pairs
+   * @param output_begin Beginning of the output sequence indicating whether each pair is present
+   * @param pair_equal The binary function to compare input pair and slot content for equality
+   * @param stream CUDA stream used for contains
+   */
+  template <typename InputIt, typename OutputIt, typename PairEqual = thrust::equal_to<value_type>>
```

Comment:
`pair_*` functions should never default the equality comparator as there is no point in using such APIs if one does not specify a custom equality function.